docs/source/torch.quantization.quantize_fx.rst

```diff
@@ -0,0 +1,24 @@
+.. _torch_quantization_quantize_fx:
```

Comment:
should we rename this to torch.ao.quantization.quantize_fx.rst?

Reply:
I was thinking that we should avoid any communications about `torch.ao` in docs until there is an OSS facing post with the details, etc.  I.e once the migration phase 2 is complete, we would update all the docs after that.  But, I'm flexible.  cc @z-a-f .

Thankfully sphinx is able to link to the docblocks in `torch.ao`, even though we are exposing the old locations before the migration.